spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe QueueItem, :type => :model do
```

Comment:
I think this `:type => :mode` could be shaved off, as you are in `spec/models` RSpec should be able to infer the type based on the file location.